src/components/ProductCard.tsx

```diff
@@ -87,71 +82,53 @@ const ProductCard = ({
   hideStars = false,
 }: ProductCardProps) => {
   const { t } = useTranslation("common")
-  const DESCRIPTION_STYLES: TextProps = {
-    opacity: 0.8,
-    fontSize: "sm",
-    mb: 2,
-    lineHeight: "140%",
-  }
 
   return (
     <Flex
-      flexDirection="column"
-      justifyContent="space-between"
-      color="text"
-      background="searchBackground"
-      boxShadow="0px 14px 66px rgba(0, 0, 0, 0.07)"
-      borderRadius="base"
-      border="1px"
-      borderColor="lightBorder"
-      textDecoration="none"
-      _hover={{
-        transition: "transform 0.1s",
-        transform: "scale(1.02)",
-      }}
+      className={cn(
+        "text-text bg-searchBackground flex-col justify-between",
+        "shadow-product-card",
```

Comment:
@nloureiro this card has a few custom shadows, Im thinking that perhaps we can simply remove them or replace them by one of the default TW shadows like `shadow-md` or `shadow-lg`. LMK what do you think.
You can see these cards used in https://ethereum.org/en/developers/local-environment/

Reply:
I will update the box shadow styles once I have more clarity. I have addressed the rest of your comments.